haystack/components/generators/hugging_face_api.py

```diff
@@ -27,15 +27,17 @@
 @component
 class HuggingFaceAPIGenerator:
     """
-    A Generator component that uses Hugging Face APIs to generate text.
+    Generates text using Hugging Face APIs.
 
-    This component can be used to generate text using different Hugging Face APIs:
+    Use it with the following Hugging Face APIs:
     - [Free Serverless Inference API]((https://huggingface.co/inference-api)
     - [Paid Inference Endpoints](https://huggingface.co/inference-endpoints)
     - [Self-hosted Text Generation Inference](https://github.com/huggingface/text-generation-inference)
 
+    ### Example usage
```

Comment:
```suggestion
    ### Usage examples
```

Just for uniform with other pages